app/models/post.rb

```diff
@@ -1,3 +1,13 @@
 class Post < ApplicationRecord
-  belongs_to :author
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+  has_many :comments
+  has_many :likes
+
+  def update_user_posts_counter
+    author.update(posts_count: author.posts.count)
+  end
+
```

Comment:
- [ ] Great work so far but it would be nice if you add this method with after_save callback to increment the posts_count automatically 🤔 You have already done that with comments_counter and likes_counter so it would be great if you increment your posts_count automatically 👍

You can make it like below ⬇️

`after_save :update_user_posts_counter`

Reply:
well resolved! ✔️